src/sasl_auth.erl

```diff
@@ -135,7 +135,7 @@ init() ->
         end,
     CanFail = application:get_env(sasl_auth,
```

Comment:
this var name means quite the opposite, please help to change it.
e.g. `FailOnLoad`